src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -3,17 +3,39 @@
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.SparkPIDController;
 
 public class ArmSubsystem extends SubsystemBase {
-     /** Creates a new FlyWheelSubsystem. */
+     /** Creates a new ArmSubsystem. */
 
     //These motors (wheels) will work in opposite of eachother so they can shoot the note out
     private CANSparkMax m_leftWheel = new CANSparkMax(96, MotorType.kBrushless);
     private CANSparkMax m_rightWheel = new CANSparkMax(95, MotorType.kBrushless);
```

Comment:
This seems like flywheel specific code, does it belong in `ArmSubsystem`?

Reply:
updated the name